buildsystem/versions.gradle

```diff
@@ -55,8 +55,9 @@ ext {
             espresso                        : '3.1.1',
             junit                           : '4.12',
             mockito                         : '2.26.0',
-            mockk                           : '1.9.3',
+            mockk                           : '1.11.0',
             powermock                       : '2.0.2',
+            robolectric                     : '4.6.1',
```

Comment:
do we need robolectric?

Reply:
Yes. To use `android.os.Bundle` in `TrackerTest`. And other android classes in future unit tests. For example `android.content.Context` etc. 

AndroidJUnit4 automatically delegates to RobolectricTestRunner when necessary.